### ISSUE TITLE
spears and arrows shouldn't be able to pierce hardsuits

### ISSUE THREAD

**Issue by JoeHammad1844** [space-wizards/space-station-14#20699]
## Description

kindof sucks when you call war ops and your entire squad gets wiped out because the crew made 20 spears and 20 potency fly amanita

this would also mean syringe guns wouldn't be completely overpowered

**Comment by Aisu9** [comment]
syringe guns is already a big no, potency for plants only dictate what you get from grinding them

**Comment by Aisu9** [comment]
crew in the other end doesn't have 50% DR again almost every damage types

**Comment by JoeHammad1844** [comment]
>  free access to instant bonus ammos/meds/explosives

not sure what you mean here because nukies need to buy those with TC, unless youre talking about the ones on the nukie outpost, in that case id like to let you know that stations also get free medkits and such mapped

**Comment by JoeHammad1844** [comment]
the anti poison medipen should probably contain stelibillin, too, because it purges amatoxin, which would be useful

**Comment by Doru991** [comment]
Yeah I agree totally on that, dylo is kinda bad tbh, and that would let nukies have a way to prevent dying from poison arrows/spears without making these weapons completely useless against them either

**Comment by JoeHammad1844** [comment]
the anti poison medipen currently uses ultravasculine

**Comment by Doru991** [comment]
In that case yeah it should have some stellibinin put into the mix, it'd be good since it's limited so nukies could afford a few slip ups but not too many

**Comment by JoeHammad1844** [comment]
i think there's only one toxin medkit on the nukie outpost, by the way

**Comment by deltanedas** [comment]
if a hardsuit can stop a bullet it should be able to stop a spear

**Comment by Aisu9** [comment]
i want to be that guy for once, hardsuits doesn't stop bullets, they don't give 25 flat pierce damage reduction
